nixos/doc/manual/development/releases.xml

```diff
@@ -98,6 +98,16 @@
       <literal>stableBranch</literal> set to false.
      </para>
     </listitem>
+    <listitem>
+     <para>
+      Remove software that we know we will not be able to support,
```

Comment:
```suggestion
      Remove attributes that we know we will not be able to support,
```

There may be attributes for specific versions of software that we won’t be able to support, which shouldn’t be included, but other, more generic attributes that refer to the same software at the time of release, that we might just change to some different, compatible software. Compare `linux_5_0` with `linux_latest`.

There’s probably a bit of a policy discussion to have around to what extent these attributes area good idea in stable, but either way I think this wording would be clearer.